dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -184,6 +185,10 @@
     @ApiModelProperty(value = "The Id of the corresponding topic on Dockstore Discuss")
     private Long topicId;
 
+    @JsonIgnore
+    @ElementCollection
```

Comment:
Oh right, should have mentioned this earlier. 
This isn't going to be used much of the time, so let's make this lazy loaded

Reply:
It defaults to LAZY

Reply:
```
    /**
     *  (Optional) Whether the collection should be lazily loaded or must be
     *  eagerly fetched.  The EAGER strategy is a requirement on
     *  the persistence provider runtime that the collection elements
     *  must be eagerly fetched.  The LAZY strategy is a hint to the
     *  persistence provider runtime.
     */
    FetchType fetch() default LAZY;
```